front/pages/i/_shortId/_slug.vue

```diff
@@ -133,6 +129,7 @@ export default {
       this.showIdeaPostedDialog = true
       // this.updateCreatedIdea(null)
     }
+    this.showExplainer = process.client && !Cookies.get('hasSeenExplainer')
```

Comment:
mounted executed ONLY at client. so its meaningless to check for process.cleint

Reply:
Removed